systems/controllers/QPCommon.cpp

```diff
@@ -502,21 +502,24 @@ int setupAndSolveQP(NewQPControllerData *pdata, std::shared_ptr<drake::lcmt_qp_c
   
   // add in body spatial equality constraints
   // VectorXd body_vdot;
-  MatrixXd orig = MatrixXd::Zero(4,1);
-  orig(3,0) = 1;
+  Vector4d orig1 = Vector4d::Zero();
+  orig1(3) = 1.0;
+  Matrix<double,3,Eigen::Dynamic> orig(3,1);
```

Comment:
The template for forwardJacDotTimesV is only instantiated for pts being Matrix<double,3,Eigen::Dynamic>, I tried to instantiate forwardJacDotTimesV for Matrix<double,3,1> but got linking error for dcrossProduct. @tkoolen ,  did you encounter this problem before?